### PR TITLE
Updated SABnzbd to 1.0.3

### PR DESCRIPTION
**Pull request by KaraokeStu**
Upgraded SABnzbd to 1.0.3 as per @Dr-Bean's request, I have made this its own pull request with a single commit

Fixes https://github.com/SynoCommunity/spksrc/issues/2191

### CODE REVIEW

File: spk/sabnzbd/Makefile [hunk 1]
```diff
@@ -14,7 +14,34 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 ADMIN_PORT = 8080
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd
-CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost."
+CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost.
```
Comment (Dr-Bean) [inline spk/sabnzbd/Makefile]: Please leave the changelog as-is (although you should update the version to what it should be, 1.0.3, not 1.0.0 at this point)

Package Center can't handle the formatting, you'd need to convert it to html for it to work properly. That's why we generally put just the minimum amount of info in there: it's a package changelog, not a software changelog. If people want to see the changes in Sabnzbd itself, they can look it up on the sabnzbd website.

**Comment by Dr-Bean** [comment]
One comment, it applies to the sabnzbd-testing package as well.
If you want, you can also add `cross/unzip` to DEPENDS, it solves the warning people currently see. I think I already added it to the -testing package, but not to the stable package yet. Otherwise we can do that in a separate commit when merging.

**Comment by KaraokeStu** [comment]
Updated commit with dependency on unzip and made changelog much shorter with HTML line breaks (as it had before)

**Comment by Dr-Bean** [comment]
The changelog isn't cumulative. This is the text I'd suggest:
```
CHANGELOG = "1. Update to 1.0.3<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.3 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.3 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost."
```

**Comment by Dr-Bean** [comment]
...and we might as well add in the unzip addition, so it looks like this:
```
CHANGELOG = "1. Update to 1.0.3<br>2. Add unzip<br>3. Warning: When you upgrade an existing 0.7.x installation, release 1.x will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.x and then use queue repair on the Status page. Some information (like the order of the queue) will be lost."
```

**Comment by KaraokeStu** [comment]
Updated to use correct HTML and remove cumulative changelog.
Made the warning red and italic ;)

**Comment by Dr-Bean** [comment]
Thanks, though I don't know that Package Center allows all HTML tags...

**Comment by KaraokeStu** [comment]
Updated as per @Dr-Bean requested

**Comment by Dr-Bean** [comment]
Multiple entries in the changelog should be numbered (somewhat documented at https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables, but it was the format previously used in the Makefile as well).
We can fix that when merging.

**Comment by Dr-Bean** [comment]
Additional changes with 13150d8e463155f29130294fe9c019bb8e673eb4.
sabnzbd 1.0.3-16 has been published on the repository.

**Pull request merged**